yt/utilities/lib/pixelization_routines.pyx

```diff
@@ -516,6 +543,9 @@ def pixelize_off_axis_cartesian(
             if mask[i,j] == 0: continue
             buff[i,j] /= mask[i,j]
 
+    if return_mask:
```

Comment:
> I would much prefer that we not do the logical not here

I'm sorry, can you clarify ?

I've been hesitant about following the logical definition of a mask or not here, but since we end up flipping all values anyway to convert to alphas, it makes sense to reverse the definition, I guess. I'll change it.

Reply:
ok, I made an attempt to change it and I broke the new image test. I'd be more confortable working on this change if we could merge the companion PR first so I don't accidentally get into a broken state.

Reply:
one coffee later: done